src/box/lua/config/applier/box_cfg.lua

```diff
@@ -355,6 +355,17 @@ local function apply(config)
     -- Set bootstrap_leader option.
     box_cfg.bootstrap_leader = configdata:bootstrap_leader()
 
+    -- Set metrics option.
+    local include = configdata:get('metrics.include', {use_default = true})
+    local exclude = configdata:get('metrics.exclude', {use_default = true})
+    local labels = configdata:get('metrics.labels', {use_default = true})
+
+    box_cfg.metrics = {
+        include = include or { 'all' },
+        exclude = exclude or { },
+        labels = labels or { alias = configdata._instance_name },
```

Comment:
Nit: It is better to use `names.instance_name` (`names` is defined above as configdata:names()`). I'm trying to keep some entities as 'more internal' and some as 'more public'.

Reply:
Ok

Reply:
Done.